scripts/lua/if_stats.lua

```diff
@@ -19,6 +19,12 @@ require "graph_utils"
 require "alert_utils"
 require "db_utils"
 
+local i18n = require "i18n"
```

Comment:
localization is done only one time in lua utils so you should avoid repeating the same code: https://github.com/ntop/ntopng/blob/dev/scripts/lua/modules/lua_utils.lua#L11